ext/opcache/Optimizer/zend_optimizer.c

```diff
@@ -531,13 +531,13 @@ int zend_optimizer_update_op2_const(zend_op_array *op_array,
 		case ZEND_FETCH_DIM_UNSET:
 		case ZEND_FETCH_LIST_R:
 		case ZEND_FETCH_LIST_W:
-			if (Z_TYPE_P(val) == IS_STRING) {
-				zend_ulong index;
-				if (ZEND_HANDLE_NUMERIC(Z_STR_P(val), index)) {
-					zval_ptr_dtor_nogc(val);
-					ZVAL_LONG(val, index);
-				}
-			}
+			// if (Z_TYPE_P(val) == IS_STRING) {
+			// 	zend_ulong index;
+			// 	if (ZEND_HANDLE_NUMERIC(Z_STR_P(val), index)) {
+			// 		zval_ptr_dtor_nogc(val);
+			// 		ZVAL_LONG(val, index);
+			// 	}
+			// }
```

Comment:
Minor note, as per our coding standards this should be a multi-line C style comment

Reply:
Oh yeah, that's just a temporary removal, didn't mean to commit that. I'll remove the commented code entirely. Thanks. 😊